airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -648,6 +622,7 @@ def __repr__(self) -> str:
     def get_local_dependency_paths(self, with_test_dependencies: bool = True) -> Set[Path]:
         dependencies_paths = []
         if self.language == ConnectorLanguage.JAVA:
+            dependencies_paths += [Path("./airbyte-cdk/java/airbyte-cdk")]
```

Comment:
This imports all of the CDK source, which isn't that big in the first place, so this should be OK. This is required when connectors are built with `useLocalCdk.= true`.